### PR TITLE
Storage: server side encryption

### PR DESCRIPTION
**Pull request by tna0y**

### CODE REVIEW

File: website/docs/r/storage_bucket.markdown [hunk 1]
```diff
@@ -101,6 +125,20 @@ The `CORS` object supports the following:
...
+
+The `server_side_encryption_configuration` object supports the following:
+
+* `rule` (Required) A single object for server-side encryption by default configuration. (documented below)
+
+The `rule` object supports the following:
+
+* `apply_server_side_encryption_by_default` (Required) A single object for setting server-side encryption by default. (documented below)
+
+The `apply_server_side_encryption_by_default` object supports the following:
```
Comment (karitra) [inline website/docs/r/storage_bucket.markdown:138]: Not an issue, but should be topic separated by new line from paragraph body?

**Review by karitra — APPROVED** [review]
LGTM.

File: website/docs/r/storage_bucket.markdown [hunk 2]
```diff
@@ -101,6 +125,22 @@ The `CORS` object supports the following:
 
 * `max_age_seconds` (Optional) Specifies time in seconds that browser can cache the response for a preflight request.
 
+* `server_side_encryption_configuration` (Optional) A configuration of server-side encryption configuration (documented below)
```
Comment (alexanderKhaustov) [inline website/docs/r/storage_bucket.markdown]: Just "A configuration of server-side encryption for the bucket"

File: website/docs/r/storage_bucket.markdown [hunk 3]
```diff
@@ -101,6 +125,22 @@ The `CORS` object supports the following:
...
+
+The `server_side_encryption_configuration` object supports the following:
+
+* `rule` (Required) A single object for server-side encryption by default configuration. (documented below)
```
Comment (alexanderKhaustov) [inline website/docs/r/storage_bucket.markdown:132]: ~~I can't tell what's the meaning of "by default configuration" here, please explain~~
Maybe change to "Rules for encryption. Only a single rule for encryption is supported for now"?

File: website/docs/r/storage_bucket.markdown [hunk 4]
```diff
@@ -101,6 +125,22 @@ The `CORS` object supports the following:
...
+
+The `rule` object supports the following:
+
+* `apply_server_side_encryption_by_default` (Required) A single object for setting server-side encryption by default. (documented below)
```
Comment (alexanderKhaustov) [inline website/docs/r/storage_bucket.markdown:136]: Is it possible to enable encryption by default and somehow disable it? If not, what's the meaning of "by default" here?

File: website/docs/r/storage_bucket.markdown (hunk 3 above)
Comment (tna0y) [reply website/docs/r/storage_bucket.markdown:132]: Although `rule` is nested there could only be one rule (in AWS as well).

File: website/docs/r/storage_bucket.markdown (hunk 4 above)
Comment (tna0y) [reply website/docs/r/storage_bucket.markdown:136]: Name is inherited from compatibility requirements for S3 API. This configuration sets encryption configuration for bucket that will apply by _default_ for all new objects uploaded to bucket.

File: website/docs/r/storage_bucket.markdown (hunk 2 above)
Comment (tna0y) [reply website/docs/r/storage_bucket.markdown]: fixed

**Comment by alexanderKhaustov** [comment]
LGTM

**Pull request merged**